orc8r/tools/ansible/roles/fluent_bit/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Add fluent bit repo to /etc/apt/sources.list
   apt_repository:
-    repo: deb https://packages.fluentbit.io/debian/stretch stretch main
+    repo: deb https://packages.fluentbit.io/{{ ansible_distribution|lower}}/{{ ansible_distribution_release }} {{ ansible_distribution_release }} main
```

Comment:
I would leave it that way or just move it to the var file as a single Variable

Reply:
this change uses ansible facts to deduce the correct repo url for fluentbit -- becomes
`deb https://packages.fluentbit.io/ubuntu/focal focal main`
should stay this way unless it causes a problem for e.g. Red Hat